.github/workflows/ci.yml

```diff
@@ -50,3 +52,22 @@ jobs:
         cd bench
         cabal bench -O0 --benchmark-options "--quick --min-duration=0 --include-first-iter"
 
+  build-freebsd:
+    runs-on: macos-latest
```

Comment:
MacOS is the most expensive system to run with [10x minute multiplier](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions). Could we use Ubuntu or Windows instead?

Reply:
Yeah, I wondered the same, but this action's [documentation](https://github.com/vmactions/freebsd-vm#under-the-hood) says that only macOS has the necessary virtualization features.

Reply:
Then we should add a comment indicating this with a link to the documentation.

Reply:
Done